ReactAndroid/src/main/java/com/facebook/react/modules/blob/BlobCollector.java

```diff
@@ -0,0 +1,25 @@
+package com.facebook.react.modules.blob;
+
+import com.facebook.react.bridge.JavaScriptContextHolder;
+import com.facebook.react.bridge.ReactContext;
+import com.facebook.soloader.SoLoader;
+
+/* package */ class BlobCollector {
+  static {
+    SoLoader.loadLibrary("reactnativeblob");
+  }
+
+  static void install(final ReactContext reactContext, final BlobModule blobModule) {
+    reactContext.runOnJSQueueThread(new Runnable() {
+      @Override
+      public void run() {
+        JavaScriptContextHolder jsContext = reactContext.getJavaScriptContextHolder();
+        synchronized (jsContext) {
```

Comment:
A comment from the internal code reviewer: we don't need the lock here since this Runnable is executed on a serial queue thread.